api/identifiers.proto

```diff
@@ -160,11 +160,6 @@ message EndDeviceVersionIdentifiers {
   string hardware_version = 3 [(validate.rules).string.max_len = 32];
   string firmware_version = 4 [(validate.rules).string.max_len = 32];
   string band_id = 5 [(validate.rules).string.max_len = 32];
-  // VendorID managed by the LoRa Alliance, as defined in TR005.
```

Comment:
Just for sanity - could you please mark these fields as reserved ? It ensures binary compatibility going forward and backward.

```
reserved 6, 7, 8; // vendor_id, vendor_profile_id, serial_number
```